pkg/products/threescale/prometheusRules.go

```diff
@@ -17,30 +16,23 @@ import (
 func (r *Reconciler) newAlertReconciler(logger l.Logger, installType string, ctx context.Context, serverClient k8sclient.Client) (resources.AlertReconciler, error) {
 	installationName := resources.InstallationNames[installType]
 
-	namespace := r.Config.GetNamespace()
-	operatorNamespace := r.Config.GetOperatorNamespace()
-	alertNamePrefix := ""
-	operatorAlertNamePrefix := ""
-
 	//clusterVersion
 	containerCpuMetric, err := metrics.GetContainerCPUMetric(ctx, serverClient, logger)
 	if err != nil {
 		return nil, err
 	}
 
-	if integreatlyv1alpha1.IsRHOAM(integreatlyv1alpha1.InstallationType(installType)) {
-		observabilityConfig, err := r.ConfigManager.ReadObservability()
-		if err != nil {
-			logger.Warning("failed to get observability config")
-			return nil, nil
-		}
-
-		namespace = observabilityConfig.GetNamespace()
-		operatorNamespace = observabilityConfig.GetNamespace()
-		alertNamePrefix = "3scale-"
-		operatorAlertNamePrefix = "3scale-operator-"
+	observabilityConfig, err := r.ConfigManager.ReadObservability()
+	if err != nil {
+		logger.Warning("failed to get observability config")
+		return nil, nil
```

Comment:
```suggestion
		return nil, err
```

Shoudl we be returning `err` here ?

Reply:
Possibly you are correct but this code is related to RHOAM and not RHMI. I feel this change would be out side the scope of this work.

Reply:
Sure, I'm good with addressing this on a later ticket